README.md

```diff
@@ -1,2 +1,30 @@
 # hpccarpentry-mini-intro
-material for our course on hpccarpentry
+material for our course on hpccarpentry.
+
+There were max 16 people last week so we can probably expect the same. This would mean a max of 3 breakout rooms.
+
+# Agenda
+## What is HPC Carpentry? (20 mins, including discussion)
+* Introduction to HPC Carpentry and what it's origins and goals are
+* What HPC Carpentry is *not*
+## (Inspired ideas from Peter go here)
+
+## Lesson development for Centres of Excellence
+FocusCoE has funded the creation of a template lesson on how to run a community code **at scale**. The idea being that such a lesson might be useful to CoEs who typically have a marquee scalable application. We chose LAMMPS as the test case because it is used heavily by the E-CAM community and development on top of LAMMPS is very common. 
+
+### Overview of lesson design (20 minutes)
+* What are the learner profiles?
+* What are the objectives?
+* What are the prerequisites?
+
+### Breakout session - Review and critique (part of) an episode of the lesson (40 minutes, 30 + 10 for discussion after)
```

Comment:
Since I know there will probably be 3 breakout rooms, I could already list the possibilities of episodes:
* https://fzj-jsc.github.io/tuning_lammps/01-why-bother-with-performance/index.html
* https://fzj-jsc.github.io/tuning_lammps/02-hardware-performance/index.html
* https://fzj-jsc.github.io/tuning_lammps/03-benchmark-and-scaling/index.html

Reply:
looks good to me. Please add this to the PR.

Reply:
OK, I am waiting for you to add these 3 breakout room ideas. Just ping me once done and I'll merge it in. I start working on the outline myself then.